source/_integrations/sws12500.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: "Sencor SWS 12500 Weather Station"
+description: "Integration of SWS 12500 weather station."
+ha_release: "2023.1.8"
+ha_category: Weather
+ha_iot_class: "Local Push"
+ha_config_flow: "true"
+ha_codeowners:
+  - '@schizza'
+ha_domain: sws12500
+ha_platforms:
+  - sensor
+---
+*Integrates your weather station seamlessly into Home Assistant.*
+
+This integration will listen for data from your station and passes them into respective sensors.
+Also you are able to push data to Windy API.
+
+## Requirements
+
+- [Sencor SWS 12500 Weather Station](https://www.sencor.cz/profesionalni-meteorologicka-stanice/sws-12500)
+- Station set to send data directly to Home Assistant
+- Open communication between station and Home Assistant
+- If you want to push data to Windy, you have to create account at [Windy](https://stations.windy.com)
+
+## Configure your station in AP mode
```

Comment:
Is this a prerequisite? If so, I would expect it to be a subheading of the above?